distro/configs/erpnext/initializer_config/price_list/insert_price_list.csv

```diff
@@ -0,0 +1,2 @@
+ID,Price List Name,Currency,Enabled,Buying,Selling,Price Not UOM Dependent
+"8264d8a8-2ca7-4edb-824b-d4af75a58f6c",Standard Discount,USD,1,0,1,0

```

Comment:
Additional q: why are the quotes required for UUIDs but not, e.g., for CIEL-style codes like `1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA`

Reply:
The import template comes with quotes for the ID column to take care of cases where the ID has spaces. Quotes aren't required for column ID If the ID has no spaces. (to be confirmed.)